src/app/modules/color-scheme/color-picker-launcher/color-picker-launcher.component.ts

```diff
@@ -24,7 +24,28 @@ export class ColorPickerLauncherComponent {
    */
   @Output() layerChange = new EventEmitter<ImageViewerLayer>();
 
+  /**
+   * Class name for the color icon
+   */
   get className(): string {
     return `color-icon ${this.layer.colorScheme.type === 'discrete' ? 'circle' : ''}`;
   }
+
+  /**
+   * Determines whether the color icon is light enough to require a border (light mode only)
+   * @param color The color hex code
+   * @returns true if brightness is above a certain threshold
+   */
+  iconLight(color: string): boolean {
```

Comment:
These two functions can be removed once you've moved the isLight/isDark onto the ImageViewerLayer class